### PR TITLE
Implement for new group

### PR DESCRIPTION
**Pull request by at-phuongdang**
Implement for new group page.

### CODE REVIEW

**Comment by at-phuongdang** [comment]
@at-hangtran 
pls review this PR. Tks!!

File: src/test/resources/features/group/NewGroup.feature [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  Scenario: I fill all data is valid, button Submit is enable
+    When I enter correct name group with text is "Android english"
+    And I open dropdown Leader
+    And I select a item in employee list
+    And I enter email of group in inputEmail with "phuong.dang@asiantech.vn" with first space is "0" and last space is "0"
+    And I enter Group Folder of group in input value with "http://google.com" with first space is "0" and last space is "0"
+    Then Button submit of new groups is display
+
+  Scenario: I fill all data is valid, button Submit is enable
```
Comment (at-hangtran) [inline src/test/resources/features/group/NewGroup.feature]: Change title of this Scenario, this is doubled with line 54

File: src/test/resources/features/group/NewGroup.feature [hunk 2]
```diff
@@ -0,0 +1,70 @@
+Feature:  Login my account to portal and open group page
```
Comment (at-hangtran) [inline src/test/resources/features/group/NewGroup.feature]: Add tag `@NewGroup` for this feature

File: src/test/java/vn/asiantech/page/group/NewGroupPage.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    @FindBy(xpath = "//input[contains(@formcontrolname,'name')]")
+    private WebElement inputNameGroup;
+    @FindBy(xpath = "//input[contains(@formcontrolname,'email')]")
+    private WebElement inputEmailGroup;
+    @FindBy(xpath = "//input[contains(@formcontrolname,'url')]")
+    private WebElement inputGroupFolder;
+    @FindBy(css = ".wrapper.wrapper-content")
+    private WebElement wapperContent;
+    @FindBy(id = "btn-create-group")
+    private WebElement btnNewGroup;
```
Comment (at-hangtran) [inline src/test/java/vn/asiantech/page/group/NewGroupPage.java]: `btnNewGroup` is not used, please remove this.

File: src/test/java/vn/asiantech/page/group/NewGroupPage.java [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+public class NewGroupPage extends BasePage<NewGroupPage> {
+
+    @FindBy(xpath = "//input[contains(@formcontrolname,'name')]")
+    private WebElement inputNameGroup;
+    @FindBy(xpath = "//input[contains(@formcontrolname,'email')]")
+    private WebElement inputEmailGroup;
+    @FindBy(xpath = "//input[contains(@formcontrolname,'url')]")
+    private WebElement inputGroupFolder;
+    @FindBy(css = ".wrapper.wrapper-content")
+    private WebElement wapperContent;
```
Comment (at-hangtran) [inline src/test/java/vn/asiantech/page/group/NewGroupPage.java]: Rename `wapperContent` to `wrapperContent`

File: src/test/java/vn/asiantech/page/group/NewGroupPage.java [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+import vn.asiantech.base.BasePage;
+
+import java.util.List;
+
+/**
+ * @author at-phuongdang
+ */
+public class NewGroupPage extends BasePage<NewGroupPage> {
+
+    @FindBy(xpath = "//input[contains(@formcontrolname,'name')]")
```
Comment (at-hangtran) [inline src/test/java/vn/asiantech/page/group/NewGroupPage.java]: Shoud use css: `@FindBy(css = "input[formcontrolname=name]")`

**Pull request merged**